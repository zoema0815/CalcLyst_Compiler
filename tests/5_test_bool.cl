int main () {
    bool true_flag;
    bool false_flag;
    true_flag = true;
    false_flag = false;
    if (true_flag) {
        printString("True flag correctly set.");
    } else {
        printString("True flag incorrectly set.");
    }
    if (false_flag) {
        printString("False flag incorrectly set.");
    } else {
        printString("False flag correctly set.");
    }
    printString("All flags are correctly set.");
    return 0;

}