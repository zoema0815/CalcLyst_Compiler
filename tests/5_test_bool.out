The flag is correctly set.
